### ISSUE TITLE
IAudioSessionManager2::GetSessionEnumerator parameter should be an output.

### ISSUE THREAD

**Issue by agausmann** [microsoft/windows-rs#529]
The [current signature](https://microsoft.github.io/windows-docs-rs/doc/bindings/windows/win32/core_audio/struct.IAudioSessionManager2.html#method.GetSessionEnumerator) of `GetSessionEnumerator`, as of 0.3.1, accepts an `&IAudioSessionEnumerator` or `&Option<IAudioSessionEnumerator>`:

```rust
pub unsafe fn GetSessionEnumerator<'a, T0__: Into<Param<'a, IAudioSessionEnumerator>>>(
    &self,
    session_enum: T0__
) -> ErrorCode
```

However, the [upstream documentation](https://docs.microsoft.com/en-us/windows/win32/api/audiopolicy/nf-audiopolicy-iaudiosessionmanager2-getsessionenumerator) says the parameter "Receives a pointer to the IAudioSessionEnumerator interface", and the [`IAudioSessionEnumerator` code example](https://docs.microsoft.com/en-us/windows/win32/api/audiopolicy/nn-audiopolicy-iaudiosessionenumerator#examples) demonstrates it being used as an out parameter. Therefore I think it should be a `*mut Option<IAudioSessionEnumerator>`

**Comment by kennykerr** [comment]
Yes, this seems like another case of this bug in the metadata: https://github.com/microsoft/win32metadata/issues/63 

This parameter should be both `[out]` and `[retval]`. Both are missing and the Rust generator doesn't know what to make of it. 

FYI @sotteson1
